lib/broadway_dashboard.ex

```diff
@@ -35,8 +38,25 @@ defmodule BroadwayDashboard do
     end
   end
 
+  defp pipelines_or_auto_discover(pipeline_config) do
+    with :auto_discover <- pipeline_config, do: Broadway.all_running()
```

Comment:
We need to run this on the remote node (and we need to be aware it may fail because the node may not have Broadway running at all!